classes/app.rb

```diff
@@ -1,164 +1,112 @@
-require_relative './book'
-require_relative './classroom'
-require_relative './person'
-require_relative './rentals'
-require_relative './student'
-require_relative './teacher'
-require 'date'
+require './app_functionality/app_books'
+require './app_functionality/app_people'
+require './app_functionality/app_rentals'
 
 class App
-  def initialize
-    @rentals = []
-    @people = []
-    @books = []
-  end
+  include AppBooks
+  include AppPeople
+  include AppRentals
+  # def list_books
+  #   if @books.length.zero?
+  #     puts 'No Books Available'
+  #   else
+  #     @books.each_with_index { |book, idx| puts "#{idx})  Book: #{book.title}, Author: #{book.author}" }
+  #   end
+  #   back_to_menu
+  # end
 
-  def console_entry_point
-    puts ''
-    puts 'Welcome to my School Library App'
-    until list_options
-      input = gets.chomp
-      if input == '7'
-        puts 'Thanks for using School Library App (^_^)'
-        break
-      end
-      option input
-    end
-  end
+  # def create_book
+  #   print 'Enter Book Title: '
+  #   title = gets.chomp
+  #   print 'Enter Book Author: '
+  #   author = gets.chomp
+  #   book = Book.new(title, author)
+  #   @books.push(book)
+  #   puts 'Book Successfully Created'
+  #   back_to_menu
+  # end
 
-  def list_options
-    puts('-------------------------')
-    puts '1 - List all books'
-    puts '2 - List all people'
-    puts '3 - Create a person'
-    puts '4 - Create a book'
-    puts '5 - Create a rental'
-    puts '6 - List all rentals of given person id'
-    puts '7 - Quit'
-    puts('-------------------------')
-    print 'Enter number of the operation: '
-  end
+  # def create_person
+  #   puts '1 - Create Student  2 - Create Teacher'
+  #   select_option
+  #   person = gets.chomp
+  #   print 'Insert age: '
+  #   age = gets.chomp
+  #   print 'Insert name: '
+  #   name = gets.chomp
+  #   person_condition(person, age, name)
+  #   puts 'Person Created Successfully'
+  #   back_to_menu
```

Comment:
**_Optional:_**
- [ ] Please remove all these commented lines to make your project more clean.

Reply:
I have removed, Thank you